src/rgw/rgw_op.cc

```diff
@@ -5517,6 +5517,7 @@ void RGWCopyObj::execute(optional_yield y)
 	   this,
 	   s->yield);
 
+  etag = "\"" + etag + "\"";
```

Comment:
we format the response in https://github.com/ceph/ceph/blob/5ab5ff46/src/rgw/rgw_rest_s3.cc#L3470-L3478. please add quotes to the etag like other requests do (ex. https://github.com/ceph/ceph/blob/5ab5ff46/src/rgw/rgw_rest_s3.cc#L1737):

```diff
-      s->formatter->dump_string("ETag", std::move(etag));
+      s->formatter->dump_format("ETag", "\"%s\"", etag.c_str());
```

Reply:
Sure. Will do the change.